addons/website_slides/controllers/main.py

```diff
@@ -506,7 +507,7 @@ def slide_get_pdf_content(self, slide):
         return response
 
     @http.route('/slides/slide/<int:slide_id>/get_image', type='http', auth="public", website=True, sitemap=False)
-    def slide_get_image(self, slide_id, field='image_medium', width=0, height=0, crop=False, avoid_if_small=False, upper_limit=False):
+    def slide_get_image(self, slide_id, field='image_medium', width=0, height=0, crop=False):
         # Protect infographics by limiting access to 256px (large) images
```

Comment:
exact same code as in `website_profile` module (only the model change), can't code be extract to a helper method?

Reply:
I agree, not only those 2 but pretty much all the places where `binary_content` is called are doing very similar things. Probably better for another task I think.